### DISCUSSION TITLE
defineEventHandler eslint error

### DISCUSSION THREAD

**Discussion by alperunal** [nuxt/nuxt#15919] [Questions]
Hi Nuxt Community,

In my server/api/[lineupId].get.ts file I'm having the following lint error:
Cannot find name 'defineEventHandler'.ts(2304)

<img width="714" alt="Screenshot 2023-01-13 at 17 56 40" src="https://user-images.githubusercontent.com/3315113/212376512-96b22f54-7a03-4a6d-b615-4ba10264afc2.png">

eslint file:
`module.exports = {
  root: true,
  env: {
    browser: true,
    node: true,
  },
  extends: [
    'plugin:nuxt/recommended',
    'plugin:vue/vue3-recommended',
  ],
  rules: {
    'comma-dangle': ['error', 'always-multiline'],
  },
}`

Any idea to get rid of this error?

**Comment by wobsoriano** [comment]
I think you're better off using the eslint Nuxt config instead - https://github.com/nuxt/eslint-config

**Comment by motea927** [comment]
Hi, you can add `import { defineEventHandler } from 'h3';`
